src/app/clusters/level-control/level-control.cpp

```diff
@@ -1224,17 +1224,19 @@ void emberAfLevelControlClusterServerInitCallback(EndpointId endpoint)
     Attributes::MaxLevel::Get(endpoint, &state->maxLevel);
 
     uint32_t featureMap;
-    if (Attributes::FeatureMap::Get(endpoint, &featureMap) == EMBER_ZCL_STATUS_SUCCESS &&
-        READBITS(featureMap, EMBER_AF_LEVEL_CONTROL_FEATURE_LIGHTING))
+    if (Attributes::FeatureMap::Get(endpoint, &featureMap) == EMBER_ZCL_STATUS_SUCCESS)
```

Comment:
Any reason this is not using `LevelControlHasFeature`?

Reply:
No. I have updated the code.